hhnk_threedi_tools/core/schematisation_builder/DAMO_exporter.py

```diff
@@ -0,0 +1,52 @@
+# %%
+"""DAMO exporter based on model extent"""
+
+import geopandas as gpd
+from hhnk_research_tools.sql_functions import (
+    database_to_gdf,
+    sql_builder_select_by_location,
+)
+from hhnk_threedi_plugin.local_settings import DATABASES
+from shapely.geometry import box
+
+
+def DAMO_exporter(model_extent, table_names, EPSG_CODE="28992"):
+    # if instead of a model_extent, polder_name is given as input.
+    # make it possible to select geometry of polder from polderclusters.gpkg
+    db_dicts = {
+        "aquaprd": DATABASES.get("aquaprd_lezen", None),
+        "bgt": DATABASES.get("bgt_lezen", None),
+    }
+
+    schema = "DAMO_W"
+
+    # make bbox --> to simply string request to DAMO db
+    bbox_model = box(*model_extent.total_bounds)
+
+    dict_gdfs_damo = {}
+    for table in table_names:
+        # Build sql to select by input polygon
+        sql = sql_builder_select_by_location(
+            schema=schema, table_name=table, epsg_code=EPSG_CODE, polygon_wkt=bbox_model, simplify=True
+        )
+        db_dict = db_dicts["aquaprd"]
+        columns = None
+
+        bbox_gdf, sql2 = database_to_gdf(db_dict=db_dict, sql=sql, columns=columns)
+
+        # clip gdf to model_extent again
+        gdf_model = gpd.clip(bbox_gdf, model_extent)
+
+        # add gdf of table export from DAMO to dictonary
+        dict_gdfs_damo[table] = gdf_model
```

Comment:
Waarom schrijven we niet naar geopackage?
We willen sowieso de export opslaan naar een DAMO.gpkg. Die moet uit de folders.py te halen zijn denk ik.
Nadeel van deze dict is ook als één van de tabellen faalt je ook alles kwijt bent.